src/custom/components/AccountDetails/Transaction.tsx

```diff
@@ -51,18 +60,202 @@ function determinePillColour(status: ActivityStatus, type: ActivityType) {
+      fill: transparent;
+      stroke: ${({ color }) => color};
+    }
+  }
+`
+
+const Summary = styled.div`
+  display: flex;
+  flex-flow: column wrap;
+  color: ${({ theme }) => theme.text1};
+
+  > b {
+    color: inherit;
+    font-weight: 500;
+    line-height: 1;
+    font-size: 15px;
+    margin: 0 0 5px;
+  }
+
+  > div {
+    display: flex;
+    flex-flow: column wrap;
+    width: 100%;
+    opacity: 0.75;
+    font-size: 13px;
+  }
+
+  > div > span {
+    display: grid;
+    grid-template-rows: 1fr;
+    grid-template-columns: 100px 1fr;
+  }
+
+  > div > span > b,
+  > div > span > i {
+    position: relative;
+    font-size: inherit;
+    font-weight: 500;
+    margin: 0;
+    color: inherit;
+    display: flex;
+    align-items: center;
+    font-style: normal;
+  }
+
+  > div > span > b {
+    padding: 0;
+    font-weight: 500;
+  }
+
+  > div > span:nth-of-type(1) > b:before,
+  > div > span:nth-of-type(2) > b:before {
+    content: '▶';
+    margin: 0 5px 0 0;
+    color: ${({ theme }) => theme.border2};
+    font-size: 8px;
+  }
+`
+
+const TransactionStatusText = styled.div`
+  margin: 0 auto 0 16px;
+  display: flex;
+  align-items: center;
+
+  &:hover {
+    text-decoration: none;
+  }
+`
+
+const StatusLabelWrapper = styled.div`
+  display: flex;
+  flex-flow: column wrap;
+  flex: 0 1 auto;
+`
+
+const StatusLabel = styled.div<{ isPending: boolean }>`
+  height: 28px;
+  width: 100px;
+  border: ${({ isPending, theme }) => isPending && `1px solid ${theme.disabled}`};
+  color: ${({ color }) => color};
+  position: relative;
+  border-radius: 4px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 12px;
+  font-weight: 600;
+
+  &::before {
+    content: '';
+    background: ${({ color, isPending }) => (isPending ? 'transparent' : color)};
+    position: absolute;
+    left: 0;
+    top: 0;
+    height: 100%;
+    width: 100%;
+    border-radius: 4px;
+    opacity: 0.1;
+  }
+
+  > svg {
+    margin: 0 5px 0 0;
+  }
+`
+
+const StatusLabelBelow = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 11px;
+  line-height: 1.1;
+  margin: 0;
+
+  ${LinkStyledButton} {
+    margin: 10px 0;
+  }
+`
+
 function getActivitySummary(params: {
   id: string
   activityData: ReturnType<typeof useActivityDescriptors>
   suffix?: string
-}): string | null {
-  const { id, activityData, suffix } = params
+  type?: ActivityType
+}) {
+  const { id, activityData, suffix, type } = params
+  const isOrder = type === ActivityType.ORDER
+  const isTransaction = type === ActivityType.TX
 
   if (!activityData) return null
 
   const { summary } = activityData
 
+  console.log(activityData)
+
   let baseSummary = summary
+  const newSummary = { from: '🤔', to: '🤔', validTo: '🤔' }
+
+  if (!suffix && baseSummary && isOrder) {
+    // Regex 'From' amount in the summary
+    // (Probably better to use the specific object keys from baseSummary instead...)
```

Comment:
As mentioned here:
> Probably better to use the specific object keys from baseSummary instead...

So perhaps better to ditch the regex here and just use the provided object keys data (e.g. `inputToken.symbol` etc.).

Reply:
Ditched